libpod/container.go

```diff
@@ -447,6 +449,17 @@ type ContainerNamedVolume struct {
 	Options []string `json:"options,omitempty"`
 }
 
+// ContainerOverlayVolume is a overlay volume that will be mounted into the
+// container. Each volume is a libpod Volume present in the state.
+type ContainerOverlayVolume struct {
+	// Destination is the absolute path where the mount will be placed in the container.
+	Dest string `json:"dest"`
+	// Source specifies the source path of the mount.
+	Source string `json:"source,omitempty"`
+	// Options are fstab style mount options.
+	Options []string `json:"options,omitempty"`
```

Comment:
Didn't we remove volumes above?

Reply:
Sorry, remove options* above

Reply:
Are you suggesting changing the comment and variable name to option, or removing the Options field?